exchanges/okx/okx_types.go

```diff
@@ -1922,31 +1922,31 @@ type BillsDetailResponse struct {
-	SpotRoleType                   string       `json:"spotRoleType"`
-	SpotTraderInsts                string       `json:"spotTraderInsts"`
-	OptionalTradingAuth            string       `json:"opAuth"` // Whether the optional trading was activated 0: not activate 1: activated
-	KYCLevel                       string       `json:"kycLv"`
-	Label                          string       `json:"label"`
-	IP                             string       `json:"ip"`
-	Permission                     string       `json:"perm"`
-	DiscountType                   string       `json:"discountType"`
-	LiquidationGear                string       `json:"liquidationGear"`
-	EnableSpotBorrow               bool         `json:"enableSpotBorrow"`
-	SpotBorrowAutoRepay            bool         `json:"spotBorrowAutoRepay"`
+	UID                            string   `json:"uid"`
+	MainUID                        string   `json:"mainUid"`
+	AccountSelfTradePreventionMode string   `json:"acctStpMode"`
+	AccountLevel                   string   `json:"acctLv"`     // 1: Simple 2: Single-currency margin 3: Multi-currency margin 4：Portfolio margin
+	AutoLoan                       bool     `json:"autoLoan"`   // Whether to borrow coins automatically true: borrow coins automatically false: not borrow coins automatically
+	ContractIsolatedMode           string   `json:"ctIsoMode"`  // Contract isolated margin trading settings automatic：Auto transfers autonomy：Manual transfers
+	GreeksType                     string   `json:"greeksType"` // Current display type of Greeks PA: Greeks in coins BS: Black-Scholes Greeks in dollars
+	Level                          string   `json:"level"`      // The user level of the current real trading volume on the platform, e.g lv1
+	LevelTemporary                 string   `json:"levelTmp"`
+	MarginIsolatedMode             string   `json:"mgnIsoMode"` // Margin isolated margin trading settings automatic：Auto transfers autonomy：Manual transfers
+	PositionMode                   string   `json:"posMode"`
+	SpotOffsetType                 string   `json:"spotOffsetType"`
+	RoleType                       string   `json:"roleType"`
+	TraderInsts                    []string `json:"traderInsts"`
+	SpotRoleType                   string   `json:"spotRoleType"`
+	SpotTraderInsts                []string `json:"spotTraderInsts"`
+	OptionalTradingAuth            string   `json:"opAuth"` // Whether the optional trading was activated 0: not activate 1: activated
+	KYCLevel                       string   `json:"kycLv"`
+	Label                          string   `json:"label"`
+	IP                             string   `json:"ip"`
+	Permission                     string   `json:"perm"`
+	DiscountType                   string   `json:"discountType"`
+	LiquidationGear                string   `json:"liquidationGear"`
+	EnableSpotBorrow               bool     `json:"enableSpotBorrow"`
+	SpotBorrowAutoRepay            bool     `json:"spotBorrowAutoRepay"`
```

Comment:
Can you please add in field Type types.Number `json:"type"`

<img width="470" alt="image" src="https://github.com/user-attachments/assets/87cb0c69-7948-48c9-8a52-e48403eba24b" />